agent/lib_aws_sdk_php.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Functions relating to instrumenting AWS-SDK-PHP.
+ */
+#ifndef LIB_AWS_SDK_PHP_HDR
+#define LIB_AWS_SDK_PHP_HDR
+
+#define PHP_AWS_SDK_SERVICE_NAME_METRIC_PREFIX \
+  "Supportability/PHP/AWS/Services/"
+#define MAX_METRIC_NAME_LEN 256
```

Comment:
Is this for any metric sent? Perhaps it would be better in `axiom/util_metrics.h`?

Reply:
The specs say it, but in looking at code, it's not explicitly handled elsewhere, so in order not to impact code elsewhere, it's only used here.